pixart.js

```diff
@@ -1 +1,21 @@
+var button = document.querySelector("button");
+var brushBox = document.querySelector(".brush");
+var colorBox = document.querySelector("#color-field");
 
+button.addEventListener( "click", function() {
+  event.preventDefault();
+  colorBox.setAttribute("value", "");
+  brushBox.style.background = colorBox.value;
+  paintColor = colorBox.value;
```

Comment:
This variable isn't used anywhere. It probably should be used on line 19 instead of re-reading from the color input box. (which you erased on line 7 above)